airbyte-webapp/src/packages/cloud/views/layout/SideBar/CloudSideBar.tsx

```diff
@@ -0,0 +1,143 @@
+import { faSlack } from "@fortawesome/free-brands-svg-icons";
+import { faCircleQuestion, faDesktop, faEnvelope } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { NavLink } from "react-router-dom";
+
+import { CreditsIcon } from "components/icons/CreditsIcon";
+import { DocsIcon } from "components/icons/DocsIcon";
+import Indicator from "components/Indicator";
+import { DropdownMenu, DropdownMenuOptionType } from "components/ui/DropdownMenu";
+import { Text } from "components/ui/Text";
+
+import { FeatureItem, IfFeatureEnabled } from "hooks/services/Feature";
+import { useCurrentWorkspace } from "hooks/services/useWorkspace";
+import { CloudRoutes } from "packages/cloud/cloudRoutePaths";
+import { useIntercom } from "packages/cloud/services/thirdParty/intercom";
+import { useGetCloudWorkspace } from "packages/cloud/services/workspaces/CloudWorkspacesService";
+import { RoutePaths } from "pages/routePaths";
+import { links } from "utils/links";
+import ChatIcon from "views/layout/SideBar/components/ChatIcon";
+import SettingsIcon from "views/layout/SideBar/components/SettingsIcon";
+import StatusIcon from "views/layout/SideBar/components/StatusIcon";
+import { NotificationIndicator } from "views/layout/SideBar/NotificationIndicator";
+import { SideBar, useCalculateSidebarStyles } from "views/layout/SideBar/SideBar";
+
+import styles from "./CloudSideBar.module.scss";
+import { LOW_BALANCE_CREDIT_TRESHOLD } from "../../credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint";
+import { WorkspacePopout } from "../../workspaces/WorkspacePopout";
```

Comment:
nit: I prefer making import paths explicit. This makes it easier to know where something comes from, and makes moving this file easier, since there are no relative paths.

```suggestion
import { LOW_BALANCE_CREDIT_TRESHOLD } from "packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint";
import { WorkspacePopout } from "packages/cloud/views/workspaces/WorkspacePopout";
```